content/ja/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Node
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Nodeは、以前には `ミニオン` としても知られていた、Kubernetesにおけるワーカーマシンです。1つのNodeはクラスターの性質にもよりますが、1つのVMまたは物理的なマシンです。各Nodeには[Pod](/docs/concepts/workloads/pods/pod/)を動かすために必要なサービスが含まれており、マスターコンポーネントによって管理されています。Node上のサービスには[コンテナランタイム](/docs/concepts/overview/components/#node-components)、kubelet、kube-proxyが含まれています。詳細については、設計ドキュメントの[Kubernetes Node](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md#the-kubernetes-node)セクションをご覧ください。
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Nodeのステータス
+
+Nodeのステータスには以下のような情報が含まれます:
+
+* [Addresses](#addresses)
+* [Conditions](#condition)
+* [CapacityとAllocatable](#capacity)
+* [Info](#info)
+
+Nodeのステータスや、Nodeに関するその他の詳細は、下記のコマンドを使うことで表示できます:
+```shell
+kubectl describe node <insert-node-name-here>
+```
+各セクションについては、下記で説明します。
+
+### Addresses
```

Comment:
“Addresses” is relevant as that's shown in the output from `kubectl describe`; should the documentation also provide a translation?

Reply:
yeah same as conditions